index.html

```diff
@@ -1025,10 +1025,10 @@ <h3>Concrete Lifecycle Example</h3>
 
         <p class="note">
 The examples above are unsecured. Implementers that are interested in
-understanding more about securing Verifiable Credentials can see the
-specifications Securing Verifiable Credentials using JOSE and COSE
-[[VC-JOSE-COSE]] and Verifiable Credential Data Integrity [[VC-DATA-INTEGRITY]]
-and the "Proofs" section of the Verifiable Credential Specifications Directory
+understanding more about securing <a>verifiable credentials</a> can see the
+specifications [[[VC-JOSE-COSE]]]
+[[VC-JOSE-COSE]] and [[[VC-DATA-INTEGRITY]]] [[VC-DATA-INTEGRITY]]
+and the "Proofs" section of the [[[VC-SPECS]]]
```

Comment:
Why add the triple brackets? We don't do this anywhere else?

Reply:
Oooh, I didn't know about this trick, either. Thank you, @TallTed -- we should apply this in many other places in the document.